webpack.config.js

```diff
@@ -122,6 +122,7 @@ const webpackConfig = {
 		],
 		alias: Object.assign(
 			{
+				'components/': '@automattic/simple-components/',
```

Comment:
I'm not confident this is working correctly.

* I don't see that these directories are symlinked (in the testing instructions).
* I'm fiddling with `node_modules/@automattic/simple-components/button/index.jsx` and `client/components/button/index.jsx`. The only changes I see are the ones from `client`.

Am I doing something wrong?